eclipse-pde/eclipse.pde
# Skip the method PluginsTab::performApply until after activation

# What it does

Fixes https://github.com/eclipse-pde/eclipse.pde/issues/1250 (a regression introduced by #1233) by skipping the whole method `PluginsTab::performApply` before the _Plugins_ tab was activated. The reason for this is that this method changes some values in the `configuration` it receives as a parameter and it does it too early. If the method is called **after** having completely activated the tab then the configuration already has the proper values and they will **not** be overwritten.

# Bug description
When a _Launch configuration_ that has a tab called _Plugins_ (_e.g._ an _Eclipse Application_  or a _JUnit Plug-In Test_) is opened and the _Plugins_ tab is selected, the default values are wrong (see screenshots in https://github.com/eclipse-pde/eclipse.pde/issues/1250#issuecomment-2074775823)

## How to reproduce
* Launch a _JUnit Plugin Test_ with _Right click > Run As... > JUnit Plugin Test_
* Open the _Run configurations_ dialog and select that run configuration that was just created
* Switch to the _Plugins_ tab --> You'll see that the usual default values for **Default Start Level** (`4`) and **Default Auto-Start** (`false`) will be replaced by `1` and `true`, like this:

![325209441-c43e1345-675d-40aa-9aff-4c628c2e78b2](https://github.com/eclipse-pde/eclipse.pde/assets/2205684/77e48c61-b7e8-4a34-a8be-5c958143c52d)

## After applying this PR
* The default values should be `4` and `false` again

![325209240-92438081-ae90-4ac5-88f3-706c626344ef](https://github.com/eclipse-pde/eclipse.pde/assets/2205684/62a8b780-7863-4e16-9128-294d974188c3)

DISCLAIMER: the screenshots are from this https://github.com/eclipse-pde/eclipse.pde/issues/1250#issuecomment-2074775823
